File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.java
import java.util.Scanner;
class Solution {
    public int solution(int num, int k) {
        int answer = 1;
        String m=String.valueOf(k);
        String s=String.valueOf(num);
        String[] str=s.split("");
        for(int i=0; i<str.length; i++){
            if(str[i].equals(m)){
                return answer;
            }else
                answer++;
        }
        if(answer==str.length+1) return -1;
        return answer;
    }
}

File: 프로그래머스/unrated/120904. 숫자 찾기/숫자 찾기.swift
import Foundation

func solution(_ num:Int, _ k:Int) -> Int {
    
    var arr = Array(String(num).compactMap{ Int(String($0))})
    var answer = -1
    for i in 0..<arr.count {
        if arr[i] == k { return i+1}
    }
    return answer
}